### PR TITLE
WR#249863: Revised installation instructions in Readme.md

### PR DESCRIPTION
**Pull request by suankan**
Updated installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,26 +57,36 @@ Never give your bot user write capabilities.**
...
 
 # Testing
 
-You can test it from the CLI using curl, see this example:
+##Test basic authentication with curl
 
-https://github.com/CatalystIT-AU/moodle-auth_basic#curl-example
+Example in bash:
+```
+curl -c /tmp/cookies -v -L --user moodlebot:moodlebot http://cqu.ubox001.com/course/view.php?id=3
```
Comment (brendanheywood) [inline README.md]: please remove cqu.ubox001.com and make it generic

File: README.md [hunk 2]
```diff
@@ -82,14 +99,11 @@ and
 
 # Travis CI integration
```
Comment (brendanheywood) [inline README.md]: Why do we even have this section?

File: README.md (hunk 1 above)
Comment (suankan) [inline README.md]: I changed it to your_moodle_website.com

File: README.md (hunk 2 above)
Comment (suankan) [inline README.md]: No point to have it, I removed it.

**Pull request merged**